lib/minitest/speed.rb

```diff
@@ -24,16 +24,20 @@ module Minitest::Speed
 
   @@max_teardown_time = 1.0
 
+  ##
+  # Default way of getting the current time.
+  @@clock_time_method = proc { Minitest.clock_time }
```

Comment:
I know this is just following the pattern given... but something in my gut says that a class var isn't appropriate here... The intent of the class var for timings was to allow for trees of tests to have their own shared timeout values... but I don't think the timing method is going to need to change per test class, is it? At least, I can't think of a scenario.

Reply:
What do you think of this?

```diff
--- old/lib/minitest/speed.rb
+++ new/lib/minitest/speed.rb
@@ -24,16 +24,25 @@
 
   @@max_teardown_time = 1.0
 
+  mc = class << self; self; end
+
+  ##
+  # Default way of getting the current time.
+
+  mc.attr_accessor :clock_time_method
+
+  self.clock_time_method = -> { Minitest.clock_time }
+
```

Reply:
Sure, sounds good to me! Pushed the changes.